[MIG][11.0] asdcust: Migrate module to v11

luisg123v opened the pull request:
This commit migrates the project to v11. The following changes were
performed:
- Updated Odoo version from 10.0 -> 11.0
- Updated Python version from 2.7 -> 3.5
- Updated PostgreSQL version to 9.6
-Added missing new lines add end of file
- Removed encoding headers from all .py files

luisg123v commented:
@ruiztulio 
Do you agree with these changes? If you do, could you create a new branch 11.0 in this project, so this PR can target it?

CC @umiphos

ruiztulio commented on .travis.yml:
```diff
@@ -7,6 +7,7 @@ cache:
     - $HOME/.cache/pip
 
 addons:
+  postgresql: "9.5"
```
@luisg123v 9.6

luisg123v replied:
@ruiztulio 
Done. Also changed the base branch and edited the commit message to reflect the PostgreSQL version

ruiztulio replied:
@luisg123v did you pushed the changes here? because I still see 9.5

luisg123v replied:
@ruiztulio 
You're right. I pushed to vauxoo-dev, as I use to do, but in this case it's a fork in my user.

ruiztulio reviewed (commented):
@luisg123v The branch 11.0 is created now

ruiztulio commented on README.md:
```diff
@@ -1,6 +1,6 @@
+[![Build Status](https://travis-ci.com/Vauxoo/asdcust.svg?token=rYtuFox23ptYNEopUvKo&branch=11.0)](https://travis-ci.com/Vauxoo/asdcust)
+[![Coverage Status](https://coveralls.io/repos/github/Vauxoo/asdcust/badge.svg?branch=11.0&t=4G2q8J)](https://coveralls.io/github/Vauxoo/asdcust?branch=11.0)
+[![Runbot Status](http://runbot.vauxoo.com/runbot/badge/93/11.0.svg)](http://runbot.vauxoo.com/runbot/repo/git-github-com-vauxoo-asdcust-git-93)
 [![Translation status](http://weblate.vauxoo.com/widgets/asdcust-10-0/-/svg-badge.svg)](http://weblate.vauxoo.com/engage/asdcust-10-0/?utm_source=widget)
```
@luisg123v 11.0 too (we are not using it, but just in case)

luisg123v replied:
Done! I had searched "10.0", but that was "10-0"

luisg123v committed:
[MIG][11.0] asdcust: Migrate module to v11

This commit migrates the project to v11. The following changes were
performed:
- Updated Odoo version from 10.0 -> 11.0
- Updated Python version from 2.7 -> 3.5
- Updated PostgreSQL version to 9.6
-Added missing new lines add end of file
- Removed encoding headers from all .py files
